frontend/src/components/regions/AgileBoard/AgileCard/Presentation.js

```diff
@@ -158,7 +158,7 @@ export default ({
         <Typography variant="caption" className={classes.title}>
           {card.contentTypeNice} {card.projectSubmissionTypeNice}
         </Typography>
-        {checkIfCardIsInReviewColumn({ card }) ? (
+        {checkIfCardIsInReviewColumn({ card }) && (authUser.email !== viewedUser.email) ? (
```

Comment:
generally, it's better to compare people by id rather than email.  This works fine but using id is better practice